addons/mail/static/tests/new/web/chatter_tests.js

```diff
@@ -592,3 +592,37 @@ QUnit.test(
         assert.containsOnce(target, ".o-mail-message-body:contains(Creating a new record...)");
     }
 );
+
+QUnit.test(
+    "should not display subject when subject is the same as the default subject",
+    async function (assert) {
+        const pyEnv = await startServer();
+        const fakeId = pyEnv["res.fake"].create({ name: "Salutations, voyageur" });
+        pyEnv["mail.message"].create({
+            body: "not empty",
+            model: "res.fake",
+            res_id: fakeId,
+            subject: "Custom Default Subject", // default subject for res.fake, set on the model
+        });
+        const { openFormView } = await start();
+        await openFormView("res.fake", fakeId);
+        assert.containsNone(target, ".o-mail-message-subject");
```

Comment:
Prefer not assert on presence on classname that are not component. This doesn't make the test robust enough.
This is even more important when you assert non-existence.